spec/Translator_spec.js

```diff
@@ -1,7 +1,113 @@
 var Translator = require('../src/Translator')
 
 describe('Translator', function() {
-  it('should fail', function(console.log, data_object) {
-    expect(true).toBe(false);
+  var outputMock;
+
+  beforeEach(function() {
+    outputMock = {
+      print: function() {}
+    };
+
+    spyOn(outputMock, 'print');
+  })
+
+  it('should convert one diagnostics data entry to something that is human readable', function() {
+    var diagnosticsData = {
+      diagnosticEntries: {
+        0: 'C7,04,00,00,EB,27,00,00,49,2A,00,00,7E,2B,00,00,F0,46,00,00,F8,28,00,00,6A,29,00,00,61,0D,00,00,49,2A,00,00,47,71,00,00,4A,71,00,00,47,71,00,00,43,1C,00,00,B5,02,00,00,'
+      }
+    };
+
+    var translator = Translator(outputMock);
+    translator.translate(diagnosticsData);
+
+    var expected = '' +
```

Comment:
I decided that I'm actually going to convert the data to JSON. I think that this way we can reuse these modules in the server and JSON will probably be easier to work with when we are trying to make it pretty.